src/transfer/stores/mockHeadless.ts

```diff
@@ -0,0 +1,62 @@
+import { observable } from "mobx";
+import { IHeadlessStore } from "./headless";
+
+export function sleep(ms: number): Promise<void> {
+  return new Promise((resolve) => setTimeout(resolve, ms));
+}
```

Comment:
You don't need to reimplement this; it's already in `utils.ts`.

Reply:
Yes, it is implemented in `utils.ts`. but `utils.ts` have electron-depend import. So we cannot use `utils.ts` if we can want to use pure js. It is not intend, we need refactor about this.